### PR TITLE
102 ux subnav

### PR DESCRIPTION
**Pull request by antonyisoardi**
Inclusion of ux-subnav

Need to check <ux-subnavtitle>. Not particularly necessary if title can be passed as an attribute of <ux-subnav>.

### CODE REVIEW

File: src/components/ux-subnav/ux-subnav.feature [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Feature: ux-subnav test suite
+
+  Scenario: Loading ux-subnav
+    Given I have loaded component "ux-subnav" with use case "UseCase"
```
Comment (ScottMaclure) [inline src/components/ux-subnav/ux-subnav.feature:4]: Q) "UseCase.json" doesn't exist?

Comment (nathf) [inline src/components/ux-subnav/ux-subnav.feature:4]: the wing task needs to be updated, this is the default `.feature` file.

File: src/components/ux-subnavitem/manifest.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (eneo5541) [inline src/components/ux-subnavitem/manifest.json:1]: It's not really necessary to include all the boilerplate for a subcomponent like this, is it? Just the handlebars and the js would suffice?

File: src/components/ux-subnavtitle/ux-subnavtitle.hbs [hunk 3]
```diff
@@ -0,0 +1 @@
+<dt>{{>content}}:</dt>
```
Comment (eneo5541) [inline src/components/ux-subnavtitle/ux-subnavtitle.hbs:1]: This subcomponent isn't called or used anywhere. I assume it's meant to be used if the subnav data has a title property?

**Comment by nathf** [comment]
Closing this in favour of #177

**Pull request closed without merging**